lib/puppetserver/ca/actions/clean.rb

```diff
@@ -0,0 +1,159 @@
+require 'puppetserver/ca/utils/parse'
+require 'puppetserver/ca/utils/http_client'
+require 'puppetserver/ca/utils/file_system'
+require 'puppetserver/ca/config/puppet'
+require 'puppetserver/ca/actions/revoke'
+
+require 'optparse'
+require 'json'
+
+module Puppetserver
+  module Ca
+    module Actions
+      class Clean
+
+        include Puppetserver::Ca::Utils::Parse
```

Comment:
by including the whole module you have access to the methods defined in it w/o having to namespace them.

I think we should just include the `Puppetserver::Ca::Utils` module, we can then directly refer to constants defined within it (eg we can just use `Parse.parse_with_errors` below)